src/main.rs

```diff
@@ -1,3 +1,5 @@
+mod name_gen;
+
 fn main() {
     println!("Hello, world!");
```

Comment:
You're probably getting a dead code warning. Just for fun make it print out `generate_simple_name("Odyssey")`!